packs/data/spell-effects.db/spell-effect-lay-on-hands.json

```diff
@@ -17,10 +18,26 @@
         },
         "rules": [
             {
+                "hideIfDisabled": true,
                 "key": "FlatModifier",
+                "predicate": [
+                    "self:mode:living"
```

Comment:
This should predicate on what the target is rather than the caster

Reply:
Both also need to pivot on negative healing.

Reply:
How exactly does negative healing interact with it? The healing would be affected, but that's taken care of by the spell variants. How does that relate to the AC bonus/penalty?

I imagine the predicates would be `living` OR NOT `negative healing`, and `undead` OR `negative healing`?